scheduler/match.go

```diff
@@ -47,10 +52,54 @@ func MemoryAvailable(v float64) ogle.Matcher {
 	return &resourceMatcher{"mem", v}
 }
 
+func (m *attributeMatcher) Matches(o interface{}) (err error) {
+	offer := o.(*mesos.Offer)
+	matched := int(0)
+
+	for _, constraint := range m.SlaveConstraints {
```

Comment:
I wasn't super familiar with the oogle interface for the AllOf matcher and whether it allowed nesting or I would need to do some kind of splat in the `createMatcher` which is why I opted for this approach.